app/controllers/concerns/remember_device_concern.rb

```diff
@@ -0,0 +1,26 @@
+module RememberDeviceConcern
+  extend ActiveSupport::Concern
+
+  def save_remember_device_preference
+    return unless params[:remember_device] == 'true'
+    cookies.encrypted[:remember_device] = RememberDeviceCookie.new(
+      user_id: current_user.id,
+      created_at: Time.zone.now
```

Comment:
Should we also set an expiration for the cookie equal to `Figaro.env.remember_device_expiration_days.to_i` from `Time.zone.now`?

Reply:
🤔, yeah good call